proposals/wac-ucr/index.bs

```diff
@@ -42,5 +42,111 @@ This is where we explain the use case.
 Definitions {#definitions}
```

Comment:
Why  is the UCR defining terminology instead of referring to where they are actually defined?

The UCR should have no knowledge about the things that's being defined here. When this is defined, it is reverse-engineering the UCR where it should instead work from its own needs to come up with the requirements.

The criteria in the WAC spec will backreference the requirements in UCR.